lib/mix/tasks/absinthe.schema.json.ex

```diff
@@ -48,7 +48,6 @@ defmodule Mix.Tasks.Absinthe.Schema.Json do
   @introspection_graphql Path.join([:code.priv_dir(:absinthe), "graphql", "introspection.graphql"])
 
   def run(argv) do
-    Mix.Task.run("app.start", [])
```

Comment:
I think we still probably want an `Application.ensure_all_started(:absinthe)` here.

Reply:
this also works too 👍